docs/application/tizen-studio/index.md

```diff
@@ -1,45 +1,45 @@
+  You may need to install additional packages to develop applications for various platforms.
 
 - [Native Tools](native-tools/index.md)
 
-  Learn about the available native application development tools. Native applications are developed using the C programming language. You can create an application project, edit the code to meet your needs, and run the application on the emulator or a target device.
+  Native applications are developed using the C programming language. You can create an application project, edit the code to meet your needs, and run the application on the emulator or a target device.
 
 - [Web Tools](web-tools/index.md)
 
-  Learn about the available Web application development tools. Web applications are developed using the JavaScript, HTML, and CSS languages. You can create an application project, edit the code to meet your needs, and run the application on the emulator, simulator, or a target device.
+  Web applications are developed using JavaScript, HTML, and CSS languages. You can create an application project, edit the code to meet your needs, and run the application on the emulator, simulator, or a target device.
 
 - [Platform Tools](platform-tools/overview.md)
 
-  Learn about the platform development environment that helps you develop preloaded platform applications, platform libraries, and device drivers in Ubuntu.
+  Platform development environment helps you to develop preloaded platform applications, platform libraries, and device drivers in Ubuntu.
 
 - [Tizen Studio for RT](rt-ide/overview.md)
 
-  Learn about the Tizen Studio for RT and how you can develop lightweight RTOS-based applications.
+  Tizen Studio for RT helps you to develop lightweight RTOS-based applications.
 
 - [Configurable SDK](configurable-sdk/configurable-sdk.md)
 
-  Learn about using the Tizen Studio to develop applications in a customizable Tizen platform environment.
+  Configurable SDK in Tizen Studio helps you to develop applications in a customizable Tizen platform environment.
 
 
-The Tizen Studio contains many tools familiar from the old Tizen SDK, but with some improvements. The following list describes the main changes:
+The Tizen Studio contains many familiar tools from the old Tizen SDK, but with some improvements. The following list describes the main changes:
 
 
 - Installer and Package Manager
 
-  These tools allow you to install, update, and maintain the Tizen Studio. You can select and install the platforms and profiles you need, and receive regular updates. The tools provide you with accurate information so you can configure the development environment easily.
+  These tools allow you to install, update, and maintain the Tizen Studio. You can select and install the platforms and profiles you need, and receive regular updates. The tools provide you with accurate information and you can configure the development environment easily:
```

Comment:
I don't know why you changed a period  to a colon. I think a period is okay.

Reply:
I know when there is "the following", "as follow", or "as follows", we are supposed to use a colon.